promotion.md

```diff
@@ -259,6 +224,74 @@ or `const` item and refer to that.
 *Dynamic check.* The Miri engine could dynamically check this by ensuring that
 the result of computing a promoted is a value that does not need dropping.
 
+### Access to a `const` or `static`
+
+When accessing a `const` in a promotable context, its value anyway gets computed
+at compile-time, so we do not have to check the initializer.  However, the
+restrictions described above still apply for the *result* of the promoted
+computation: in particular, it must be a valid `const` (i.e., it cannot
+introduce interior mutability) and it must not require dropping.
+
+For instance, while the previous example was not legal, the following would be:
```

Comment:
The previous example is now at the end of the document.

Reply:
Hm. IMO it would be better to use something else as example here, e.g. a `const fn` that is not `rustc_promotable`. That seems less artificial than this. Any proposals?

Reply:
Nothing that isn't contrived comes to mind. Even just a call to a const fn would be better than the current example IMO.

Reply:
Done.